### PR TITLE
Document release process

### PR DESCRIPTION
**Pull request by jotak**
(also remove default readme files)

Adressing #7 

A follow-up for more automation is planned: #33

### CODE REVIEW

**Comment by jotak** [comment]
@josejulio @israel-hdez as you have some expertise in releasing do you mind having a look at this?

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Release process
+
+This is currently a manual process:
+
+- Bump versions in `web/pf3/package.json` and `web/pf4/package.json`
```
Comment (josejulio) [inline RELEASE.md]: Where does the `go` part takes its version?

Comment (jotak) [reply RELEASE.md]: It automatically takes it from the git tag

**Comment by israel-hdez** [comment]
Sorry @jotak . I completely forgot about this :(

**Pull request merged**